include/HeyPublisherXML/HeyPublisherXML.class.php

```diff
@@ -222,13 +110,15 @@ public function send($path,$post) {
   * This is the only method which does not call prepare_request_xml - as we have a custom <account> section
   */
   function authenticate($user) {
+    global $hp_config;
     $return = false;
     // authentication is based upon username, password, and token
     $xml_ops = array(
       'token'         => HEYPUB_SVC_TOKEN_VALUE,
       # no htmlentities here, otherwise " becomes %quote; and cronks the seo name
-      'publishername' => stripslashes($this->get_config_option('name')),
-      'url'           => htmlentities(stripslashes($this->get_config_option('url'))),
+      'publishername' => stripslashes($hp_config->get_config_option('name')),
+      // TODO: Fix this!! url is now a nested var
+      'url'           => htmlentities(stripslashes($hp_config->get_config_option('url'))),
```

Comment:
Need to validate, but this should not break as this var will not have value upon plugin initiation, which is where authenticate gets called.